SwiftWisdom/Rx/Rx+Extensions.swift

```diff
@@ -23,17 +23,15 @@ infix operator <-> : Binding
 infix operator <- : Binding
 infix operator >>> : Disposing
 
-public func <- <T>(property: AnyObserver<T>, variable: Observable<T>) -> Disposable {
-    return variable
-        .observeOn(MainScheduler.instance)
-        .bindTo(property)
+public func <- <T: ObserverType, O: ObservableType>(property: T, variable: O) -> Disposable where T.E == O.E {
```

Comment:
It could be because they're not in an extension on ObserverType.  I will look into moving them there now.